lsp-mode.el

```diff
@@ -3992,6 +3998,20 @@ in that particular folder."
            (lsp--on-change-debounce buffer)
            (lsp--on-idle buffer)))))))
 
+(defun lsp-unconfig-buffer ()
+  (when lsp-modeline-code-actions-mode
```

Comment:
are those checks needed or we can directly try to turn it off? At least I have never seen a code that checks if the mode is on before disabling it.

Reply:
I think it's needed, it will run the `t` block in the minor mode initialization. So better guard for it do not run again.